osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -240,6 +257,74 @@ private void load(TextureStore textures)
             }
         }
 
+        // using string literals as there's no proper processing for LocalizeStrings yet
+        public class SupporterRequiredDrawable : CompositeDrawable
+        {
+            private OsuSpriteText supporterRequiredText;
+
+            public SupporterRequiredDrawable()
+            {
+                RelativeSizeAxes = Axes.X;
+                Height = 225;
+                Alpha = 0;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(TextureStore textures)
+            {
+                AddInternal(new FillFlowContainer
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    RelativeSizeAxes = Axes.Y,
+                    AutoSizeAxes = Axes.X,
+                    Direction = FillDirection.Horizontal,
+                    Children = new[]
+                    {
+                        new Sprite
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            RelativeSizeAxes = Axes.Both,
+                            FillMode = FillMode.Fit,
+                            Texture = textures.Get(@"Online/supporter-required"),
+                        },
+                        supporterRequiredText = new OsuSpriteText
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            Font = OsuFont.GetFont(size: 16),
+                            Colour = Colour4.White,
+                            Margin = new MarginPadding { Bottom = 10 },
+                        },
+                        createSupporterTagLink(),
+                    }
+                });
+            }
+
+            public void UpdateText(bool playedFilter, bool rankFilter)
+            {
+                List<string> filters = new List<string>();
+                if (playedFilter) filters.Add(BeatmapsStrings.ListingSearchFiltersPlayed.ToString());
+                if (rankFilter) filters.Add(BeatmapsStrings.ListingSearchFiltersRank.ToString());
+                supporterRequiredText.Text = BeatmapsStrings.ListingSearchSupporterFilterQuoteDefault(string.Join(" and ", filters), "").ToString();
+            }
+
+            private Drawable createSupporterTagLink()
+            {
+                LinkFlowContainer supporterTagLink = new LinkFlowContainer
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    AutoSizeAxes = Axes.Both,
+                    Margin = new MarginPadding { Bottom = 10 },
+                };
+
+                supporterTagLink.AddLink(BeatmapsStrings.ListingSearchSupporterFilterQuoteLinkText.ToString(), "https://osu.ppy.sh/store/products/supporter-tag");
```

Comment:
* By convention, untranslatable string literals use the raw string notation (`@"https://osu.ppy.sh/store/products/supporter-tag"`).
* The URL of the website changes depending on debug/release configuration, so the `https://osu.ppy.sh` domain should not be hardcoded like here. Use `IAPIProvider.WebsiteRootUrl` to handle this properly.